pkg/eventer/event_recorder.go

```diff
@@ -0,0 +1,44 @@
+package eventer
```

Comment:
I am not sure why there is a wrapper around this.
The recorder could be inside controller and all could use c.Event().